Dockerfile

```diff
@@ -48,9 +48,9 @@ USER postgres
 
 # Pre-build extension dependencies
 RUN cd ../ && cargo pgx new promscale && cd promscale
-COPY Cargo.* Makefile /build/promscale/
+COPY Cargo.* Makefile create-upgrade-symlinks.sh /build/promscale/
```

Comment:
Why do we need to copy over or use the create-upgrade-symlinks.sh? Isn't this just about caching Rust dependencies? But create-upgrade-symlinks.sh is about the SQL not the Rust stuff and the SQL isn't even copied over from the container at this point. So I am confused.

Reply:
Ya you're right. It doesn't need to be copied over at this point yet. I've moved it to `package` only.

Reply:
I'm not sure if that answered your question adequately though, please follow up if something's still unclear...